### PR TITLE
Downgrade to latest stable symfony/framework-bundle

### PR DESCRIPTION
**Pull request by hallabro**
symfony/framework-bundle 2.8 belongs to the symfony 2.8 release which is
still in development. Packages making use SpBowerBundle would before
this commit require a dependency which should not be used for production
quality code.

composer.lock has also been added to the repository in order to help
developers test this package and ensuring they are using the same
versions. Please see https://getcomposer.org/doc/02-libraries.md#lock-file
for more information.

### CODE REVIEW

**Comment by hallabro** [comment]
The failed builds are not a regression from this PR as far as I can see. The builds for older versions of symfony fail because of incompatible versions of ´symfony/framework-bundle`.

File: composer.json [hunk 1]
```diff
@@ -18,7 +18,7 @@
     ],
     "require": {
         "php": ">=5.3.3",
-        "symfony/framework-bundle": "2.8.x-dev",
```
Comment (Spea) [inline composer.json]: Seems like i accidentally changed it to 2.8.x-dev -.-

File: composer.json [hunk 2]
```diff
@@ -18,7 +18,7 @@
...
+        "symfony/framework-bundle": "2.7.*@dev",
```
Comment (Spea) [inline composer.json]: Please change that line to `~2.1`

File: .gitignore [hunk 3]
```diff
@@ -1,5 +1,4 @@
 vendor/
-composer.lock
```
Comment (Spea) [inline .gitignore]: That entry should not be removed

**Comment by Spea** [comment]
Please also remove the `composer.lock` file from this PR. The file was removed in PR #82 and should not be added again

**Comment by hallabro** [comment]
Yeah sure but there are advantages to it being there. As an example, it would be impossible for a third developer now to download this repository and knowingly run the tests against the same versions of a package that we know works.

I think it's a better practice to manually update the dependencies when you feel like it (i.e. running `composer update`, verifying that it works correctly, and commit the changes).

**Comment by Spea** [comment]
The composer.json should make sure that the "minimal" requirements are met so the bundle can be used. Also the composer.lock will not have any effect on the projects that depend on it. And last but not least there is travis, which makes sure, that this bundle will work on every symfony version, starting from 2.1.

Some time ago I managed a composer.lock for this bundle, but it was quite annoying to do so and I personally think it is better without one.

File: composer.json (hunk 2 above)
Comment (Spea) [inline composer.json]: Can you change this? I would merge it directly then.

**Comment by hallabro** [comment]
While I do not agree I reverted the lock file. It would be good to have this fix merged as per [https://github.com/Spea/SpBowerBundle/issues/128](issue 128).

**Comment by Spea** [comment]
I would merge it if you will change the version for `symfony/framework-bundle` in the composer.json as mentioned in https://github.com/Spea/SpBowerBundle/pull/129#issuecomment-113871367

**Comment by hallabro** [comment]
Weird, I must have missed that notification somehow.

**Pull request merged**